intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+\chapter{Using the HPC-UGent web portal}
+\label{ch:web_portal}
+
+\href{https://openondemand.org/}{\strong{Open OnDemand}} (or \texttt{OoD} for short)
+is a web portal to access HPC resources.
+
+You can use it to connect to the HPC-UGent infrastructure via:
+
+\begin{center}\url{https://login.hpc.ugent.be}\end{center}
+
+You do not need to install any additional software to use the web portal, a standard web browser like Firefox,
+Google Chrome or Safari (macOS) is sufficient. In addition, no SSH key pair is required to connect
+to your VSC account.
+
+Through the web portal, you can:
+\begin{itemize}
+    \item browse through your files \& directories, and inspect, manage or change them;
+    \item consult active jobs (across all HPC-UGent Tier-2 clusters);
+    \item submit new jobs to the HPC-UGent Tier-2 clusters, either from existing job scripts or from job templates;
+    \item start an interactive graphical user environment (a.k.a. desktop), either on the login nodes or on a cluster workernode;
+    \item open a terminal session directly in your web browser;
+\end{itemize}
+
+More detailed information is available below, as well as in the \href{https://osc.github.io/ood-documentation/master/}{Open OnDemand documentation}.
+
+An Open OnDemand walkthrough video is available on YouTube \href{https://www.youtube.com/watch?v=4-w-4wjlnPk}{here}.
+
+\section*{Pilot access}
+
+\strong{The HPC-UGent web portal is currently available for all VSC accounts,
+but there may be a couple of small problems that we still need to resolve.}
+
+\strong{If you notice anything that doesn't work as expected or could be improved,
+please contact \hpcinfo.}
+
+\strong{In addition, you \emph{must} be connected to the HPC-UGent network
+(via \href{https://helpdesk.ugent.be/vpn/en/}{VPN} if necessary)
+in order to use the HPC-UGent web portal.}
+
+\section{Login}
+
+When visiting the HPC-UGent web portal via \url{https://login.hpc.ugent.be},
+you will be automatically logged in via the VSC accountpage, which wll in turn
+authenticate you via your institutional login service (for UGent also known as CAS).
+
+You may need to enter your UGent login name and password in the standard CAS login page if you are not logged in yet.
+
+\subsection{First login}
+
+The first time you visit \url{https://login.hpc.ugent.be} permission will be requested to let Open OnDemand
+access the VSC accountpage to authenticate you, as shown in this screenshot below:
```

Comment:
the authentication part is already passed at this point. this is the user allwoing 3rd party like the webportal access some personal info. in this case, the personal info is the vscid, accoutn status, login shell and insitute name. as it is not dsiplayed in the request, you might point this out here. if needed, i can limit this to the vsc id alone.

Reply:
OK, clarified.